app/layouts/DashboardLayout/BalanceIndicator.view/BalanceIndicator.view.tsx

```diff
@@ -1,25 +1,51 @@
 import React from 'react';
 import type { FC } from 'react';
 
-import { Box, Button, makeStyles, Typography, useMediaQuery } from '@material-ui/core';
+import {
+  Box,
+  Button,
+  makeStyles,
+  Typography,
+  useMediaQuery,
+  Select,
+  MenuItem,
+  ListItemIcon,
+} from '@material-ui/core';
+import MonetizationOnOutlinedIcon from '@material-ui/icons/MonetizationOn';
 import { useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
 
 import { MOBNumberFormat } from '../../../components';
 import { MOBIcon } from '../../../components/icons';
+import { TokenIds } from '../../../constants/app';
 import { GOLD_LIGHT } from '../../../constants/colors';
+import { ReduxStoreState } from '../../../redux/reducers/reducers';
+import { setTokenId } from '../../../redux/services';
 import { Theme } from '../../../theme';
```

Comment:
future pr, but adding pathing aliases wold make this a lot cleaner a la https://hello-js.com/articles/path-aliases-in-react-application/

Reply:
honestly, we should just use absolute imports. Makes copying and pasting imports so much easier